kratos/includes/variables.h

```diff
@@ -34,6 +34,7 @@
 #include "containers/periodic_variables_container.h"
 #include "utilities/table_stream_utility.h"
 #include "utilities/adjoint_extensions.h"
+#include "utilities/derivatives_extension.h"
```

Comment:
We should try to think of a way to avoid these includes here (also for adjoint extensions). Basically all of kratos depends on this file.

Reply:
I tried using forward declarations, but with kratos::make_unique it gives incomplete type error. Therefore had to leave it like this.